### PR TITLE
ADD: Bugsnag on watchOS app

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/vjvbhesw64pmoo6zeupyxzlt7i?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/299)

File: .github/workflows/build-ios-release-testflight.yml [hunk 1]
```diff
@@ -19,6 +19,7 @@ jobs:
       latest_commit_message: ${{ steps.get_latest_commit_message.outputs.commit_message }}
     env:
       APPLE_ID: ${{ secrets.APPLE_ID }}  # Setting the environment variable
+      BUGSNAG_API_KEY: ${{ secrets.BUGSNAG_API_KEY }}  # Add this line
```
Comment (Overtorment) [inline .github/workflows/build-ios-release-testflight.yml:22]: Don't we have this api key publicly available and hardcoded in js code..?
Should be safe to just hardcode it here as well

**Review by Overtorment — CHANGES_REQUESTED** [review]

File: .github/workflows/build-ios-release-testflight.yml (hunk 1 above)
Comment (marcosrdz) [reply .github/workflows/build-ios-release-testflight.yml:22]: Will do

Comment (marcosrdz) [reply .github/workflows/build-ios-release-testflight.yml]: Done

**Comment by Overtorment** [comment]
conflicts

**Comment by marcosrdz** [comment]
> conflicts

solved

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/hpx5lp4lgeksokemyytjamqknu?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/301)

File: ios/BlueWalletWatch Extension/ExtensionDelegate.swift [hunk 2]
```diff
@@ -8,12 +8,18 @@
...
+import Bugsnag
 
 class ExtensionDelegate: NSObject, WKExtensionDelegate {
   
+  let groupUserDefaults = UserDefaults(suiteName: UserDefaultsGroupKey.GroupName.rawValue)
+
   func applicationDidFinishLaunching() {
     scheduleNextReload()
     updatePreferredFiatCurrency()
+    if let isDoNotTrackEnabled = groupUserDefaults?.bool(forKey: "donottrack"), !isDoNotTrackEnabled {
```
Comment (Overtorment) [inline ios/BlueWalletWatch Extension/ExtensionDelegate.swift:20]: you added it to applewatch  code? eh, lets add it to ios app as well, my idea with js-onlyimplementation didnt work.
perhaps in another PR

**Review by Overtorment — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/66h7nruee43xialkdtphegqg3i?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/302)

File: ios/BlueWalletWatch Extension/ExtensionDelegate.swift (hunk 2 above)
Comment (marcosrdz) [reply ios/BlueWalletWatch Extension/ExtensionDelegate.swift:20]: We already have crash reports that seem pre-Metro bundler. Its super crucial to have it on the native side

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/drxn5l45utwuhrzf6omqeob6fy?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/306)

File: ios/BlueWalletWatch Extension/ExtensionDelegate.swift (hunk 2 above)
Comment (Overtorment) [reply ios/BlueWalletWatch Extension/ExtensionDelegate.swift:20]: Yeah I agree

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/msplhznadxxduenq7fqgpqolou?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/310)

**Pull request merged**